clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -55,10 +57,12 @@
 
     private AtomicBoolean wakeup;
 
+    private AtomicBoolean rebalanced = new AtomicBoolean(false);
```

Comment:
This is initialized with false, however once set true, it is never set to true again. Is the usage as you intended?